en-EN.js

```diff
@@ -366,38 +366,38 @@ module.exports = {
-            agreement_confirm_button: "Accepter",
-            agreement_reject_button: "Rejeter",
-            agreement_reject_message: "**Nous n'enregistrerons pas vos préférences.**\n> Si vous avez davantage de questions sur la manière dont nous disposons de vos **données**, vous pouvez rejoindre notre **Serveur Discord**. *Autrement, si vous changez d'avis, vous pouvez retaper la commande !*",
+            agreement_message: "**Important : ** before you edit your card's arrangement, you must accept we save your preferences on *all your servers*. This data will **not be deleted** if you leave this server or delete your account and will be **public**, hence why this message's important./If you **accept** the data collection and storage, press **'Accept'** below.",
+            agreement_confirm_button: "Accept",
+            agreement_reject_button: "Refuse",
+            agreement_reject_message: "**We won't save your preferences.**\n> if you have more questions about how we manage your **data**, you can join our **Discord server**. *Otherwise, if you change your mind, you can type the command again !*",
             agreement_reject_easteregg_gif: "https://tenor.com/view/dujardin-jean-daccord-comme-ca-gif-10162502",
-            main_embed_description: "**Modifiez** la disposition des éléments statistiques sur votre carte. *Sélectionnez un élément dans le menu déroulant pour le modifier !*",
-            buttons_save: "Sauvegarder",
-            main_embed_fields_name: "Ligne {n} :",
-            main_embed_select_displayed: "Affiché :",
-            main_embed_select_placeholder: "Sélectionner un élément",
-            select_type_embed_description: "**Très bien !** Choisissez quel type d'élément afficher dans l'emplacement **#{n}**.",
-            select_type_embed_cancelled: "**Temps mort !** La sélection de l'élément a été annulée.",
-            select_content_description: "**Très bien !** Choisissez quel type de statistique vous souhaiteriez afficher dans la boîte `{name}`.",
-            select_content_menu_locked_description: "Sélectionnez pour acheter cette statistique.",
+            main_embed_description: "**Edit** the arrangement of the statistic elements on your card. *Select an element in the dropdown menu to edit it!*",
+            buttons_save: "Save",
+            main_embed_fields_name: "Line {n} :",
+            main_embed_select_displayed: "Displayed :",
+            main_embed_select_placeholder: "Select an element",
+            select_type_embed_description: "**Alright!** Select what element type to display in this location **#{n}**.",
```

Comment:
`Select what kind of element you want to..` ne serrait pas un peu moins verbeux ?